README.md

```diff
@@ -70,6 +70,7 @@ $ npx vitest
 - [Unit Testing](./test/core)
 - [Vue Component Testing](./test/vue)
 - [React Component Testing](./test/react)
+- [React Mock Service Worker with Testing Library Testing](./test/msw-testing-lib-react)
```

Comment:
How about renamed the folder to `react-testing-lib-msw`?

Reply:
:clap: :clap: